Mathmagician.Tests/EvenNumberTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class EvenNumberTests
+    {
+        [TestMethod]
+        public void MakeSureYouCanMakeEvenNumber()
+        {
+            //Arrange
+
+            //Act
+            EvenNumber usereven = new EvenNumber();
+
+            //Assert
+            Assert.IsNotNull(usereven);
+        }
+
+
+        [TestMethod]
+        public void CheckIfUserNumberIsEven()
+        {
+            EvenNumber neweven = new EvenNumber();
+
+            neweven.usereven = 7;
+
+            Assert.IsFalse(neweven.usereven % 2 == 0);
+
+        }
+
+        [TestMethod]
+        public void CheckIfAdding2Works()
+        {
+            EvenNumber neweven = new EvenNumber();
+
+            neweven.usereven = 4;
+
+            Assert.AreEqual(6, neweven.usereven + neweven.counter);
```

Comment:
@Odigene12 Your assert has too much logic in it. You shouldn't be adding anything in your assert statement. Isn't this a method you should test?